src/dejsonlz4.c

```diff
@@ -61,7 +62,19 @@ void *file_to_mem(const char *fname, size_t *out_size)
 {
     size_t fsize = 0;
     void *rv = 0;
-    FILE *f = fopen(fname, "rb");
+    FILE *f;
+    if (fname) {
+        f = fopen(fname, "rb");
+    } else {
+#ifdef _WIN32
+        /* Try changing stdout to binary mode, or else it messes with EOL chars.
```

Comment:
`Try changing stdin ...`

Also, as obvious addition as it might seem (the binary mode), did you test that it works?

Reply:
I no longer work on Windows, but I used the idiom all the time when I was and it worked like a charm.